### PR TITLE
Feat agenda

### PR DESCRIPTION
**Pull request by mixcas**
Moves the event-item into a partial
Add archive for events aka Agenda
add very few styles just to make it look like it make sense

### CODE REVIEW

File: archive-event.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="col s2 feed-category">
+      <a href="<?php echo esc_url( $events_archive_link ); ?>">Agenda</a>
+    </div>
+
+  <?php
+  $post_index = 0;
+  while( have_posts() ) {
+    the_post();
+
+    get_template_part('partials/event_item');
```
Comment (varffs) [inline archive-event.php]: I think this file is named wrong and should be a dash not underscore

File: partials/event_item.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+$venue = get_post_meta( $post->ID, '_igv_event_address', true );
+?>
+
+<article <?php post_class('col s7 event-item'); ?> id="post-<?php the_ID(); ?>">
+
+  <a href="<?php the_permalink() ?>">
+    <div class="event-date col s3 u-align-center">
+    <p class="month"><?php echo $month; ?></p>
+    <p class="day"><?php echo $day; ?></p>
```
Comment (varffs) [inline partials/event_item.php]: this echos 0 if no date is set

**Comment by varffs** [comment]
I think to pull this it should be totally finished. Otherwise it isn't really a feature. We are missing the subtitle line of venue name [actually needs another metabox for that we forgot it]

File: archive-event.php (hunk 1 above)
Comment (mixcas) [inline archive-event.php]: u rite

**Comment by mixcas** [comment]
@varffs we got a `event_address` (Lugar), which it would be better as `event_venue`. Also posts and events share the sub-title metabox

**Comment by mixcas** [comment]
nvm, i think we need a diff field for `event_venue`…

got some questions…

**Comment by mixcas** [comment]
@varffs this is ready

**Pull request merged**